src/prefect/orion/models/deployments.py

```diff
@@ -423,12 +427,7 @@ async def _generate_scheduled_flow_runs(
     # retrieve the deployment
     deployment = await session.get(db.Deployment, deployment_id)
 
-    if (
-        not deployment
-        or not deployment.manifest_path
-        or not deployment.schedule
-        or not deployment.is_schedule_active
-    ):
+    if not deployment or not deployment.schedule or not deployment.is_schedule_active:
```

Comment:
would this start scheduling deprecated deployments again?

Reply:
Yea it could if folks still have old deployments with active schedules lying around -- we could look at `created` timestamp or something, or just documentation?